css-contain-2/Overview.bs

```diff
@@ -1994,6 +1994,37 @@ Suppressing An Element's Contents Entirely: the 'content-visibility' property {#
 		When the element becomes disconnected, the element's [=proximity to the viewport=] becomes [=not determined=].
 	</div>
 
+	<div>
+		All elements have a <dfn>currently relevant to the user</dfn> flag, initially false.
+	</div>
+
+	<div algorithm="update content relevancy for a document">
+		To <dfn export>update content relevancy for a document</dfn> given a document <var>doc</var>, and a boolean <var>isFirstIteration</var>:
+
+		1. Let <var>changed</var> be false.
+		1. For each <var>element</var> [=connected=] to doc with ''auto'' used value of ''content-visibility''
+			1. Determine proximity to the viewport for <var>element</var>.
+			1. If <var>element</var> is not [=currently relevant to the user=] and <var>element</var> is [=relevant to the user=], then:
+				2. Set <var>changed</var> to true.
+				2. Set <var>element</var>'s [=currently relevant to the user=] flag to true.
+			1. If <var>isFirstIteration</var> is false, then [=continue=].
```

Comment:
Because of the comment below, it's unclear to me the role isFirstIteration is playing here. `changed` is set in either case

Reply:
The important bit is that if `isFirstIteration` is false, then non-initial determination is not supposed to happen (this is what implements the semantics you were talking about in your other comments). I think I just botched the condition  in line 2007 (it's supposed to deal only with the initial proximity determination).

I'll fix, thanks for the review.